common/src/main/res/values/strings.xml

```diff
@@ -1302,4 +1302,16 @@
     <string name="basic_sensor_name_car_range_remaining">Car range remaining</string>
     <string name="sensor_description_car_range_remaining">The range remaining for the car in meters</string>
     <string name="sensor_description_health_connect_heart_rate">Last recorded heart rate in beats per minute from Health Connect</string>
+    <string name="basic_sensor_name_body_fat">Body fat</string>
+    <string name="sensor_description_body_fat">Last recorded body fat percentage from Health Connect</string>
+    <string name="basic_sensor_name_distance">Distance</string>
+    <string name="sensor_description_distance">Get distance traveled in meters from Health Connect</string>
+    <string name="basic_sensor_name_elevation_gained">Elevation gained</string>
+    <string name="sensor_description_elevation_gained">Get elevation gained in meters from Health Connect</string>
+    <string name="basic_sensor_name_floors_climbed">Floors climbed</string>
+    <string name="sensor_description_floors_climbed">Get floors climbed from Health Connect</string>
+    <string name="basic_sensor_name_sleep_duration">Sleep duration</string>
+    <string name="sensor_description_sleep_duration">Get sleep duration in minutes from Health Connect</string>
```

Comment:
Should this mention _last recorded_ sleep duration?

Reply:
For this I am not sure because its an aggregate number

Reply:
So it is minutes of sleep today and not the x minutes I slept yesterday evening??

Reply:
logic has been changed to pull from the last record instead of aggregate, latest update now reflects this